scripts/rest_client.py

```diff
@@ -124,7 +124,11 @@ def delete_lock(self, lock_id):
         :param lock_id: ID for the lock to delete
         :return: json object
         """
-        return self._delete(f"locks/{lock_id}")
+        if not self.locks().get(lock_id):
+            logger.warning("Lock id not found!")
+            return None
+        else:
+            return self._delete(f"locks/{lock_id}")
```

Comment:
I'm not entirely sure about this.

i think i'd prefer to have the script simply fail (which it does now - or eventually catch the 404 error that occurs if we attempt to delete a lock that doesn't exist - rather than first querying ALL locks (which could be 1000nds) just to find out if we actually can delete a lock or not.